src/Stepper.js

```diff
@@ -62,7 +65,7 @@ class Stepper {
                 }
             }
 
-            doing(1 - easing(time));
+            doing(getNow(time));
```

Comment:
-I think that we may need more options relate to getNow() so extract that but i don't know what is need. 

-I added zero(0) to intend to mean to show animation is working between 0 and 1 in code.